data/lib/utils/constant/firebase_storage_constant.dart

```diff
@@ -17,4 +17,10 @@ class StorageConst {
     required String imageName,
   }) =>
       "$rootDirectory/$userId/support_attachment_images/$imageName";
+
+  static String tournamentProfileUploadPath({
+    required String userId,
+    required String tournamentId,
+  }) =>
+      "$rootDirectory/$userId/tournament_profile_images/$tournamentId";
```

Comment:
how about below hierarchy:
- Profile: $rootDirectory/$userId/tournament_images/$tournamentId/**profile**
- Banner: $rootDirectory/$userId/tournament_images/$tournamentId/**banner**

or

- Banner: $rootDirectory/$userId/tournament_images/$tournamentId/**imageName** (in case, to support multiple banners)